docs/index.md

```diff
@@ -1,4 +1,164 @@
+`/query`.
+
+# Data Schema
+![](QueryServiceDataSchema.svg)
+
+Because there are often multiple, slightly different versions of a bundle or file, `bundles_all_versions` and 
+`files_all_versions` contain all versions of the bundles and files. There are also derived view tables `files` and `bundles`
+which only contain the latest version of each bundle or file. 
+
+The metadata itself is available in `files.body` as a jsonb data object. The structure of that document is dependent on the `dcp_schema_type`. The schemas for each schema type can be found [here](https://schema.humancellatlas.org/a). 
+Its also possible to pull the full jsonb for a file of a particular schema type and explore it in your text editor to better understand what data it contains and how it is formatted.
+For example to get the jsonb associated with a cell line
+```postgresql
+SELECT body FROM files WHERE dcp_schema_type='cell_line';
+``` 
+or
+```postgresql
+SELECT body FROM cell_line;
+```
+To simplify queries, the view tables for each schema type contain only the most recent version of each metadata file as the schema of the metadata may change between versions 
+
+Possible schema_types include 
+`cell_line`,
+`cell_suspension`,
+`differentiation_protocol`,
+`dissociation_protocol`,
+`donor_organism`,
+`ipsc_induction_protocol`,
+`library_preparation_protocol`,
+`links`,
+`organoid`,
+`process`,
+`project`,
+`sequence_file`,
+`sequencing_protocol`,
+`specimen_from_organism`,
+`supplementary_file`,
+`analysis_file`,
+`analysis_process`,
+`analysis_protocol`,
+`collection_protocol`,
+`enrichment_protocol`,
+`biomaterial`,
+`file`,
+`image_file`,
+`imaged_specimen`,
+`imaging_preparation_protocol`,
+`imaging_protocol`
+
+
+# Example Queries
+Get a list of all of the tables
+```postgresql
+SELECT table_name
+FROM information_schema.tables
+WHERE table_type = 'BASE TABLE'
+ AND table_schema NOT IN ('pg_catalog', 'information_schema')
+ AND table_schema = 'public';
+```
+Get total number of bundles
+```postgresql
+SELECT count(*) from bundles;
+```
+Get all data for 10 bundles
+```postgresql
+SELECT * FROM bundles LIMIT 10;
+```
+Select the data for a particular bundle
+if you use the uuid this may return multiple versions of the bundle, to only get one use the fqid (which is the version concatenated to the uuid)
+```postgresql
+SELECT * FROM bundles WHERE uuid='000f989a-bea2-46cb-9ec1-b4de8c292931'
+```
+## Querying the metadata body
+The metadata itself is stored in the jsonb format, for help with the syntax of querying jsonb check out this [cheat sheet](https://hackernoon.com/how-to-query-jsonb-beginner-sheet-cheat-4da3aa5082a3).
+
+Get a list of submissions for a particular submitter
+```postgresql
+SELECT p.uuid, p.body->'project_core'->>'project_title' AS title
+FROM project AS p
+WHERE p.body @> '{"contributors": [{"contact_name": "Aviv,,Regev"}]}'
+   OR p.body @> '{"contributors": [{"contact_name": "Sarah,,Teichmann"}]}';
```

Comment:
error in query builder ```ERROR:  relation "project" does not exist
LINE 3: FROM project AS p```

Reply:
this is the same `create_views` issue as above